README.md

```diff
@@ -32,38 +47,45 @@ On Windows (in PowerShell):
 $env:PATH += ";$env:USERPROFILE\.poetry\bin"
 ```
 
-TEMP WORKAROUND: poetry cannot handle azure-identity package in current version (1.0.10), so install the preview version, until it's released.
+### Update Poetry
+
+This is a temporary workaround! The currently released version of Poetry
+(1.0.10) cannot handle the `azure-identity` package, so we need to install the
+preview version (1.1.0b2).
 
 ```bash
-poetry self update --preview 1.1.0a2
+poetry self update --preview 1.1.0b2
```

Comment:
I tried b2, but may not work as there are some bugs.

Reply:
I actually ran into a bug with a2 that was resolved with b2 🙃 I think it was this: https://github.com/python-poetry/poetry/issues/2769

Reply:
It looks my problem only happens on poetry self update, not on fresh installing. Will update instruction.

Reply:
Oh, good to know!